db/models.py

```diff
@@ -14,3 +14,36 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(null=True)
+    actors = models.ManyToManyField(Actor, related_name="actors")
```

Comment:
it is wrong related name.
someone will do: actor.actors and as a result will get the list of movies. For me it strange. Please read how the related name works